skimage/restoration/deconvolution.py

```diff
@@ -8,62 +8,63 @@
 
 
 def wiener(image, psf, balance, reg=None, is_real=True, clip=True):
-    r"""Wiener-Hunt deconvolution
+    r"""Restore image using Wiener-Hunt deconvolution.
 
-    Return the deconvolution with a Wiener-Hunt approach (i.e. with
-    Fourier diagonalisation).
+    Wiener-Hunt deconvolution is a restoration method which follows a Bayesian
+    approach [1]_.
 
     Parameters
     ----------
-    image : ndarray
-       Input degraded image (can be n-dimensional).
+    image : ([P, ]M, N) ndarray
+       Degraded image.
     psf : ndarray
-       Point Spread Function. This is assumed to be the impulse
-       response (input image space) if the data-type is real, or the
-       transfer function (Fourier space) if the data-type is
-       complex. There is no constraints on the shape of the impulse
+       Point spread function (PSF). Assumed to be the impulse
+       response (input image space) if the data type is real, or the
+       transfer function (Fourier or frequency space) if the data type is
+       complex. There is no constraint on the shape of the impulse
        response. The transfer function must be of shape
-       `(N1, N2, ..., ND)` if `is_real is True`,
-       `(N1, N2, ..., ND // 2 + 1)` otherwise (see `np.fft.rfftn`).
+       ``(N1, N2, ..., ND)`` if `is_real is True`,
+       ``(N1, N2, ..., ND // 2 + 1)`` otherwise (see :func:`numpy.fft.rfftn`).
     balance : float
-       The regularisation parameter value that tunes the balance
-       between the data adequacy that improve frequency restoration
-       and the prior adequacy that reduce frequency restoration (to
-       avoid noise artifacts).
+       Regularization parameter. Denoted by :math:`\lambda`: in the Notes
+       section below, its value lets you balance data adequacy (improving
+       frequency restoration) with respect to prior adequacy (reducing
+       frequency restoration and avoiding noise artifacts).
     reg : ndarray, optional
-       The regularisation operator. The Laplacian by default. It can
-       be an impulse response or a transfer function, as for the
-       psf. Shape constraint is the same as for the `psf` parameter.
+       Regularization operator. Laplacian by default. It can
+       be an impulse response or a transfer function, as for the PSF.
+       Shape constraints are the same as for `psf`.
     is_real : boolean, optional
-       True by default. Specify if ``psf`` and ``reg`` are provided
-       with hermitian hypothesis, that is only half of the frequency
```

Comment:
I removed this phrase ("Hermitian hypothesis") because I couldn't find a reference for it anywhere; you can't find it in the [1] reference paper; when you search for it online, the only results are from these docs here!!